database/database-cft.yaml

```diff
@@ -0,0 +1,107 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: Template for creating testbed database instance
+
+Parameters:
+  StaticStackName:
+    Type: String
+    Default: dev-ga4gh-testbed-database
+  ParamDBName:
+    Type: String
+    Default: testbed_db
+  ParamUsername:
+    Type: String
+    Default: testbed_writer
+  SubnetA:
```

Comment:
`SubnetA`, `SubnetB` and `SubnetC` are unused. They can be removed.

Reply:
Removed.